app/controllers/preprints/provider/moderation.js

```diff
@@ -13,6 +14,12 @@ export default Ember.Controller.extend({
         },
         pageChanged(page) {
             this.set('page', page);
+            Ember.get(this, 'metrics')
+                .trackEvent({
+                    category: 'button',
+                    action: 'click',
+                    label: `Request page ` + this.get('page') + ` of ` + this.get('status') + ` list`
```

Comment:
I think it'd be good to sort of namespace labels by page/component, like ember-preprints does: "Moderation List - Request page..."